blazor/treeview/virtualization.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: Virtualization in Blazor TreeView Component | Syncfusion
+description: Checkout and learn here all about Virtualization in Syncfusion Blazor TreeView component and much more details.
+platform: Blazor
+control: TreeView
+documentation: ug
+---
+
+# Virtualization in Blazor TreeView Component
+
+The TreeView has been provided virtualization support to improve the UI performance for a large amount of data when [EnableVirtualization](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_EnableVirtualization) is true. This feature doesn’t render out the entire data source on initial rendering. It loads the N number of items in the initial rendering and the remaining set number of items will load on each scrolling action in the TreeView container. To setup the virtualization, define the [EnableVirtualization](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_EnableVirtualization) as true and content height by [Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_Height) property.
```

Comment:
Can mention as UI Virtualization @vishwanathan3797 and initial data retrieval can be explained

Reply:
changes added.